src/classes/StatsD.class.php

```diff
@@ -192,4 +191,67 @@ public function setHost($host)
         $this->host = $host;
     }
 
+    /**
+     * @param string $operation
```

Comment:
Took a bit of jumping around function to function to work out what each one of these was doing.

It might be of value to add a desc here.